packages/orbit-components/src/ChoiceGroup/components/FilterWrapper.tsx

```diff
@@ -16,67 +13,28 @@ interface Props {
   ) => void | Promise<void>;
 }
 
-const hoverAndFocus = () => css`
-  background-color: ${({ theme }) => theme.orbit.paletteBlueLight};
-
-  ${StyledOnlyButton} {
-    visibility: visible;
-    opacity: 1;
-  }
-`;
-
-const StyledContentWrapper = styled.div<{ disabled: boolean }>`
-  ${({ disabled }) => css`
-    box-sizing: border-box;
-    width: 100%;
-    padding-left: 4px;
-    border-radius: 4px;
-    display: flex;
-    align-items: center;
-    height: ${({ theme }) => theme.orbit.heightButtonSmall};
-
-    ${StyledOnlyButton} {
-      visibility: hidden;
-      opacity: 0;
-    }
-
-    ${!disabled &&
-    css`
-      @media (hover: none) {
-        ${StyledOnlyButton} {
-          visibility: visible;
-          opacity: 0.3;
-          &:hover {
-            opacity: 1;
-          }
-        }
-      }
-
-      @media (hover) and (pointer: fine) {
-        &:hover {
-          ${hoverAndFocus};
-        }
-
-        &:focus-within {
-          ${hoverAndFocus}
-        }
-      }
-    `};
-  `}
-`;
-
-StyledContentWrapper.defaultProps = {
-  theme: defaultTheme,
-};
-
 const FilterWrapper = ({ child, children, onOnlySelection, onlySelectionText }: Props) => {
   const { value, label, disabled } = child.props;
 
   return (
-    <StyledContentWrapper disabled={disabled}>
+    <div
+      className={cx(
+        "h-form-box-small box-border flex w-full items-center rounded-[4px] pl-[4px]",
+        "[&_.orbit-choice-group-filter-wrapper]:invisible [&_.orbit-choice-group-filter-wrapper]:opacity-0",
+        !disabled &&
+          "hover:[@media(hover)_and_(pointer:fine)]:bg-blue-light focus-within:[@media(hover)_and_(pointer:fine)]:bg-blue-light group",
+      )}
+    >
       {children}
       {onOnlySelection && !disabled && (
-        <StyledOnlyButton>
+        <div
+          className={cx(
+            "orbit-choice-group-filter-wrapper",
+            "[@media(hover:none)]:!visible [@media(hover:none)]:!opacity-30 hover:[@media(hover:none)]:!opacity-100",
```

Comment:
what is `!` in Tailwind? `!important`?

Reply:
Yes, it means `!important`.

The classes with `!` represent this CSS code:

```css
@media (hover: none) {
  ${StyledOnlyButton} {
    visibility: visible;
    opacity: 0.3;
    &:hover {
      opacity: 1;
    }
  }
```

that was originally applied via the parent element (`StyledContentWrapper`) and I moved it to the Filter button for convenience. The problem is that in the parent we have:

```css
${StyledOnlyButton} {
    visibility: visible;
    opacity: 1;
  }
```

so cascading is not being applied properly.

I moved the latter style classes to the button to avoid the !important and now looks cleaner, thanks!